packages/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -716,17 +716,99 @@ describe('plugin-mercury', () => {
             .then((wsUrl) => assert.match(wsUrl, /multipleConnections/)));
       });
     });
+  });
+  describe('ping pong latency event is forwarded', () => {
```

Comment:
This tests fails when we run it with all the other tests. https://github.com/webex/webex-js-sdk/commit/b3a5cdf8bc69c5c4ba5439464b85f431d20ad21d - PR in next

Reply:
Since we have changed mercury to use mocha, this change is not required anymore.